MobileApplication/MobileApplication/Models/Item.cs

```diff
@@ -4,6 +4,15 @@ namespace MobileApplication.Models
 {
     public class Item
     {
+        public Item(string id, string text, string description)
+        {
+            Id = id;
+            Text = text;
+            Description = description;
+        }
+
+        public Item() { }
+
         public string Id { get; set; }
```

Comment:
What for This class is used?

Reply:
For the binding context in the view models